packages/mockserver-config-writer/src/translations/mockserver-config-writer.i18n.json

```diff
@@ -1,6 +1,6 @@
 {
     "questions": {
-        "pathToMock": "Path to mocked service",
-        "overwrite": "Overwrite services"
+        "pathToMock": "Path to Mocked Service",
+        "overwrite": "Overwrite Services"
```

Comment:
These are questions to the user, not sure why we should have them all capital letters?

It is similar to these texts here:
https://github.com/SAP/open-ux-tools/blob/1a1d780c7c8ca06b0a110d9c48bb1acea3d11681/packages/ui5-library-inquirer/src/translations/ui5-library-inquirer.i18n.json#L10